go/service/config.go

```diff
@@ -46,9 +47,37 @@ func (h ConfigHandler) GetCurrentStatus(_ context.Context, sessionID int) (res k
 	return
 }
 
-func getPlatformInfo() keybase1.PlatformInfo {
+func (h ConfigHandler) getOsVersion() string {
```

Comment:
we don't follow this but `get` is sort of implied and not used in most go code... up to you though..

ancronyms are all caps in go... `getOSVersion()` but if a private method and not using "get" would be `osVersion()`